src/main/java/duke/Execution.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import duke.command.*;
+import duke.exception.CommandException;
+
+public class Execution {
+    protected String fullCommand;
+    protected String command;
+    protected boolean isExit;
+
+    public Execution(String fullCommand) {
+        this.fullCommand = fullCommand;
+        this.command = Parser.command(fullCommand);
+        this.isExit = false;
+    }
+
+    private static void CheckCommand(String command) throws CommandException {
```

Comment:
I think the method name should start with a lower case letter.

Reply:
Thx for advice.